### PR TITLE
[Method References 2] LPS-167592 Convert method reference to ServiceTracker in classes using class NodeTypeDependentObjectRegistry

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:workflow

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/node/NodeExecutorRegistry.java [hunk 1]
```diff
@@ -14,62 +14,42 @@
...
  * @author Michael C. Han
  */
 @Component(service = NodeExecutorRegistry.class)
 public class NodeExecutorRegistry {
 
-	public NodeExecutor getNodeExecutor(String nodeTypeString)
-		throws KaleoDefinitionValidationException {
-
-		return _nodeExecutors.getNodeTypeDependentObjects(nodeTypeString);
+	public NodeExecutor getNodeExecutor(String nodeTypeString) {
```
Comment (tinatian) [inline modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/node/NodeExecutorRegistry.java:34]: @W2jx Please modify the parameter type from String to NodeType, then you do not need to do valueOf

Comment (W2jx) [reply modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/node/NodeExecutorRegistry.java:34]: Hi @tinatian , In classes using method `getNodeExecutor`, the parameter passed to it is `KaleoNode.getType`. We may have to use method `NodeType.valueOf` once. Do you want me to use it before passing parameters?

Comment (tinatian) [reply modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/node/NodeExecutorRegistry.java:34]: @W2jx My bad, I thought the invocations used the enum type

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/127135. Please pay attention to my changes.

Thanks.
TIna

**Comment by W2jx** [comment]
Hi @tinatian , thank you so much, these changes inspired me!

**Pull request closed without merging**